bootstrap/kubeadm/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -19,7 +19,3 @@ spec:
         ports:
         - containerPort: 8443
           name: https
-      - name: manager
-        args:
-        - "--metrics-bind-addr=127.0.0.1:8080"
-        - "--leader-elect"
```

Comment:
The upstream kubebuilder example has this, it might be good to leave it just for the sake of similarity even if it's not strictly needed.

Reply:
If I leave this I get a kustomize error because it is not able to merge args from this patch and from `bootstrap/kubeadm/config/default/manager_webhook_patch.yaml`, so I remove this one because they are not needed

Reply:
Given that we're going to enable webhooks by default, we should move all of these args in the main manager file, instead of having them around in places

Reply:
(Applies to all the other manages as well)

Reply:
done!